fluidsynth/src/sfloader/fluid_defsfont.c

```diff
@@ -3168,9 +3168,9 @@ fixup_sample (SFData * sf)
     {
       sam = (SFSample *) (p->data);
       
-      // The SoundFont 2.4 spec defines the loop start index as the first sample point of the loop
+      /* The SoundFont 2.4 spec defines the loop start index as the first sample point of the loop */
       inv_start = (sam->loopstart < sam->start) || (sam->loopstart >= sam->loopend);
-      // while loop end is the first point AFTER the last sample of the loop.
+      /* while loop end is the first point AFTER the last sample of the loop */
       inv_end = (sam->loopend > sam->end+1) || (sam->loopstart >= sam->loopend);
```

Comment:
Oh wait, fixup_sample() acutally corrects it by -1 later on. But talking about this particular line of code is still points behind sample data. _Insane_